charts/reform-scan-blob-router/values.preview.template.yaml

```diff
@@ -1,22 +1,22 @@
 java:
   secrets:
     STORAGE_ACCOUNT_NAME:
-      secretRef: storage-account-{{ .Release.Name }}-blobstorage
+      secretRef: storage-account-reform-scan
```

Comment:
I think best to keep the {{ .Release.Name }} here and just lose the -blobstorage